webapp/channels/src/components/common/hooks/useExpandOverageUsersCheck.ts

```diff
@@ -31,7 +31,10 @@ export const useExpandOverageUsersCheck = ({
 }: UseExpandOverageUsersCheckArgs) => {
     const {formatMessage} = useIntl();
     const dispatch = useDispatch();
-    const {getRequestState, is_expandable: isExpandable}: LicenseSelfServeStatusReducer = useSelector((state: GlobalState) => state.entities.cloud.subscriptionStats || {is_expandable: false, getRequestState: 'IDLE'});
+    const statsDefault: LicenseSelfServeStatusReducer = useMemo(() => {
```

Comment:
Instead of initializing `statsDefault` inside the body of the component, could you turn this into a memoized selector using `createSelector`?